src/app.rb

```diff
@@ -0,0 +1,138 @@
+require_relative 'library/book'
+require_relative 'library/rental'
+require_relative 'people/school/student'
+require_relative 'people/school/teacher'
+require 'date'
+require 'yaml'
+
+class App
+  MENU = "1 - List all books.
+2 - List all people.
+3 - Create a person.
+4 - Create a book.
+5 - Create a rental.
+6 - List all rentals for a given person id.
+7 - Quit".freeze
+  def initialize
+    @books = []
+    @people = []
+    @rentals = []
+    @date = Time.new.strftime('%Y-%m-%d')

```

Comment:
**Hint**

- Please keep in mind that you specified the initials of your instance variables of the `App` class, and then you tried to call them as class variables with the double `@@`, and not single `@`